src/TuDa.CIMS.Web/Components/AssetItemDialog.razor

```diff
@@ -0,0 +1,239 @@
+﻿@rendermode InteractiveServer
+
+@page "/modal"
+
+@* Modal Button *@
+<button class="btn btn-primary"
+        data-bs-target="#createItemDialog"
+        data-bs-toggle="modal"
+        type="button">
+    Create Item
+</button>
```

Comment:
🤔 🔧I think we should move this somewhere else and maybe create a generic model component.
Maybe some examples are written down [here](https://stackoverflow.com/questions/59256798/how-to-use-bootstrap-modal-in-blazor-client-app).